packages/lib/BaseApplication.ts

```diff
@@ -747,8 +747,10 @@ export default class BaseApplication {
 		}
 
 		if (Setting.value('firstStart')) {
-			const locale = shim.detectAndSetLocale(Setting);
-			reg.logger().info(`First start: detected locale as ${locale}`);
+			if (!Setting.value('isSubProfile')) {
```

Comment:
Thank for this fix. Would you mind adding a short comment to explain why we have this check?

Reply:
Not at all, I've added a comment to the code.